src/components/ui/formik/SingleNodeCheckbox.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { useField, useFormikContext } from 'formik';
+import { Checkbox } from '@patternfly/react-core';
+import { CheckboxFieldProps } from './types';
+import { getFieldId } from './utils';
+import HelperText from './HelperText';
+import { ClusterCreateParams } from '../../../api/types';
+
+const SingleNodeCheckbox: React.FC<CheckboxFieldProps> = ({
+  label,
+  helperText,
+  onChange,
+  validate,
+  idPostfix,
+  ...props
+}) => {
+  const [field] = useField({ name: props.name, validate });
+  const fieldId = getFieldId(props.name, 'input', idPostfix);
+
+  const { setFieldValue } = useFormikContext<ClusterCreateParams>();
+
+  return (
+    <Checkbox
+      {...field}
+      {...props}
+      id={fieldId}
+      label={label}
```

Comment:
Since this is custom field just for Single node option the label and helperText can be defined directly here and simplified. No need to pass them as props